### PR TITLE
Rami/infra(video drv)

### PR DESCRIPTION
**Pull request by ramizi**

### CODE REVIEW

File: src/components/general/templates/videoPlayer.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<video ng-show="isAvail"controls preload = "none">
```
Comment (igormas) [inline src/components/general/templates/videoPlayer.html]: Please use ng-switch

File: src/components/general/videoActivationButtonDrv.module.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
+'use strict';
+angular.module('znk.infra.general')
+       .directive('videoActivationButtonDrv',
+           ['$window', '$http', '$log', function ($window, $http, $log) {
+               return {
+                   restrict: 'E', scope: {
```
Comment (igormas) [inline src/components/general/videoActivationButtonDrv.module.js]: Align.

File: src/components/general/videoActivationButtonDrv.module.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+                       playVideo: '&ngClick'
+                   }, link: function (scope, element, attrs) {
+                       var elmCur = element[0].children[0];
+                       elmCur.poster =
```
Comment (igormas) [inline src/components/general/videoActivationButtonDrv.module.js]: What is this poster ?

File: src/components/general/videoActivationButtonDrv.module.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+                           'http://corrupt-system.de/assets/media/sintel/sintel-trailer.jpg';
+                       scope.disabledButton = true;
```
Comment (igormas) [inline src/components/general/videoActivationButtonDrv.module.js]: Who use this property ?

File: src/components/general/videoActivationButtonDrv.module.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+                       scope.analyticsLabel = '';
```
Comment (igormas) [inline src/components/general/videoActivationButtonDrv.module.js]: Who use this property ?

File: src/components/general/videoActivationButtonDrv.module.js [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+                       scope.isLoading = true;
```
Comment (igormas) [inline src/components/general/videoActivationButtonDrv.module.js]: Who use this property ?

File: dist/znk.infra.js [hunk 7]
```diff
@@ -1111,6 +1111,96 @@
...
+                   }, link: function (scope, element, attrs) {
+                       var elmCur = element[0].children[0];
+                       elmCur.poster =
+                           'http://corrupt-system.de/assets/media/sintel/sintel-trailer.jpg';
+                       scope.disabledButton = true;
+                       scope.analyticsLabel = '';
+                       scope.isLoading = true;
+                       scope.isAvail = true;
+                       scope.pathFull =
+                           (angular.isDefined(attrs.vidLib) && attrs.vidLib !== '') ?
```
Comment (igormas) [inline dist/znk.infra.js]: What is this link ?

File: src/components/general/videoActivationButtonDrv.module.js (hunk 3 above)
Comment (ramizi) [inline src/components/general/videoActivationButtonDrv.module.js]: attribute of video html5

File: src/components/general/videoActivationButtonDrv.module.js (hunk 4 above)
Comment (ramizi) [inline src/components/general/videoActivationButtonDrv.module.js]: I don't know need to ask oren before I remove it

File: src/components/general/videoActivationButtonDrv.module.js (hunk 5 above)
Comment (ramizi) [inline src/components/general/videoActivationButtonDrv.module.js]: I don't know need to ask oren before I remove it

File: src/components/general/videoActivationButtonDrv.module.js (hunk 6 above)
Comment (ramizi) [inline src/components/general/videoActivationButtonDrv.module.js]: I don't know need to ask oren before I remove it

**Pull request closed without merging**